policy/test_submount.te

```diff
@@ -0,0 +1,39 @@
+#################################
+#
+# Policy for testing NFS crosmnt
+#
+
+gen_require(`
+	attribute file_type;
+	type nfs_t;
+')
+
+# A domain that can access NFS files/directories
+type test_readnfs_t;
+domain_type(test_readnfs_t)
+unconfined_runs_test(test_readnfs_t)
+domain_obj_id_change_exemption(test_readnfs_t)
+typeattribute test_readnfs_t testdomain;
+
+# Allow execution of helper programs
+corecmd_exec_bin(test_readnfs_t)
+domain_exec_all_entry_files(test_readnfs_t)
+libs_use_ld_so(test_readnfs_t)
+libs_use_shared_libs(test_readnfs_t)
+libs_exec_ld_so(test_readnfs_t)
+libs_exec_lib_files(test_readnfs_t)
+
+# Allow this domain to be entered from sysadm domain
+miscfiles_domain_entry_test_files(test_readnfs_t)
+userdom_sysadm_entry_spec_domtrans_to(test_readnfs_t)
+corecmd_bin_entry_type(test_readnfs_t)
+sysadm_bin_spec_domtrans_to(test_readnfs_t)
+
+# Allow the domain to search all dirs so that ls works
+# no matter where testsuite is run from.
+allow test_readnfs_t file_type:dir { search getattr };
```

Comment:
This is broader than the rest of the test policy, which appears to only grant e.g. userdom_search_generic_user_home_dirs().  If you need it to be this general, then perhaps files_search_all(test_readnfs_t) is suitable as an interface call.

Reply:
You're right, I shouldn't break the consistency here, so I just used `userdom_search_generic_user_home_dirs()`.